NBitcoin/BIP174/PartiallySignedTransaction.cs

```diff
@@ -1517,12 +1506,12 @@ public PSBT CoinJoin(PSBT other)
 		/// </summary>
 		/// <param name="errors"></param>
 		/// <returns></returns>
-		public PSBT Finalize(out InvalidOperationException[] errors)
+		private PSBT Finalize(out InvalidOperationException[] errors)
```

Comment:
I was expecting this to be a public first, but it may not be a big difference

Reply:
but the public overload still throws an aggregateException, in .NET this is the way to get error codes (except if you use the Try pattern, in which case it doesn't throw, to save performance)